inventory/group_vars/co.yml

```
@@ -0,0 +1,22 @@
+---
+
+# See https://github.com/openfoodfoundation/ofn-install/wiki/Setup for more info
+checkout_zone: Colombia
+country_code: CO
+currency: COP
+locale: es_CO               # Default locale for OFN instance
+available_locales: es_CO    # Available locales for OFN instance
+language: es_MX.UTF-8       # Default language for the server
```

```suggestion
language: es_CO.UTF-8       # Default language for the server
```

I'd rather use this. Looks like bionic includes it :point_right: https://packages.ubuntu.com/focal/all/locales/filelist

---

This is the version of the configs I got directly from Colombia's syadmin... :man_shrugging:

---

As for the `.UTF-8` suffix, the package names for locales use the format `en_GB`, environment variables for selected locales on the system use the format `en_GB.utf8`, but the command to _set_ the current locale for the system uses the format `en_GB.UTF-8`.

Simple, right? :trollface: 

References [here](https://wiki.ubuntu.com/UbuntuDevelopment/Internationalisation/InternationalizationPrimer/Locales#line-31) and [here](https://wiki.ubuntu.com/UbuntuDevelopment/Internationalisation/InternationalizationPrimer/Locales#line-80).

#ridiculous